crates/bevy_render/src/camera/camera.rs

```diff
@@ -66,6 +66,31 @@ impl Default for Viewport {
     }
 }
 
+/// Settings to define a camera sub view.
+///
+/// When [`Camera::sub_camera_view`] is `Some`, only the sub-section of the
+/// image defined by `size` and `offset` (relative to the `full_size` of the
+/// whole image) is projected to the cameras viewport.
+#[derive(Debug, Clone, Copy, Reflect, PartialEq, Eq)]
+pub struct SubCameraView {
+    /// Size of the entire camera view
```

Comment:
I'd add "in pixels".

Reply:
The sub view is defined by the ratio between `full_size`, `size` and `offset`, so in this case "in pixels" would be misleading. That was however not sufficiently documented, so I have added further detail.